### PR TITLE
remove brackets

### PR DESCRIPTION
**Pull request by privatezero**
I can't actually test, but after poking around in `sh` I think this ought to take care of https://github.com/amiaopensource/vrecord/issues/619

### CODE REVIEW

**Review by retokromer — APPROVED** [review]

File: vrecord [hunk 1]
```diff
@@ -1138,8 +1138,8 @@ export MAIN_DIALOG='
     <action signal="leave-notify-event" type="refresh">VRECORD_FORM_VALIDATION</action>
     <action signal="show" type="closewindow">HOME_DIALOG</action>
     <action signal="show" type="refresh">VRECORD_OUTPUT_NAME</action>
-    <action signal="show" condition="command_is_true( [[ \"$AUDIO_DEV_CHOICE\" != *\"Blackmagic\"* ]] && echo true)">enable:AUDIO_MODE_SR_CHOICE</action>
-    <action signal="show" condition="command_is_true( [[ \"$AUDIO_DEV_CHOICE\" == *\"Blackmagic\"* ]] && echo true)">disable:AUDIO_MODE_SR_CHOICE</action>
+    <action signal="show" condition="command_is_true( [ \"$AUDIO_DEV_CHOICE\" != *\"Blackmagic\"* ] && echo true)">enable:AUDIO_MODE_SR_CHOICE</action>
+    <action signal="show" condition="command_is_true( [ \"$AUDIO_DEV_CHOICE\" = *\"Blackmagic\"* ] && echo true)">disable:AUDIO_MODE_SR_CHOICE</action>
```
Comment (dericed) [inline vrecord:1142]: This isn't working on my side. How about:

`[ \"${AUDIO_DEV_CHOICE/Blackmagic}\" != \"${AUDIO_DEV_CHOICE}\" ]`

Comment (retokromer) [reply vrecord:1142]: @dericed On which macOS version?

**Comment by dericed** [comment]
@retokromer 10.15.7, but this is run within the gtkdialog subshell

File: vrecord (hunk 1 above)
Comment (dericed) [reply vrecord:1142]: @retokromer 10.15.7, but this is run within the gtkdialog subshell

Comment (retokromer) [reply vrecord:1142]: @dericed Thank you! Sorry, I forgot that all versions of macOS I have here at home aren’t set up with the Bash version kindly provided by Apple™…

**Review by dericed — APPROVED** [review]
works for me

**Review by retokromer — APPROVED** [review]
Works also on my machines: 
- macOS 11.0 public beta 5, 10.15.7, 10.14.6 and 10.13.6 (with Bash 5)
- Debian 10.6 and 9.13; Ubuntu 20.04.1 LTS, 18.04.5 LTS and 16.04.7 LTS; Slackware 14.2; Qubes OS 4.0.3
- Windows 10: version 2004, 1909, 1903 and 1809, running Terminal

**Comment by privatezero** [comment]
closing in favor of https://github.com/amiaopensource/vrecord/compare/shell-compatibility (cleaner commit history)

**Pull request closed without merging**